game.py

```diff
@@ -20,7 +20,29 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    wrong_guesses_count = 0
+    continue_playing = True
+
+    while continue_playing:
+        letter = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
```

Comment:
While this line doesn't break PEP8 guidelines for line length, it certainly gets close (79 characters or less including indentation https://peps.python.org/pep-0008/#maximum-line-length). From the linked guide:

> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses.

Nothing is wrong here, I'm just putting it out there so it's on your radar!